core/src/CMakeLists.txt

```diff
@@ -65,6 +64,7 @@ set(_srcs
 set(_private_headers
   util/usFrameworkPrivate.h
   util/usLDAPExpr_p.h
+  util/usLog.h
```

Comment:
I suggest to rename that file to `usLog_p.h` to be consistent with the current state.